app/src/main/res/layout/fragment_suggested_edits_card_item.xml

```diff
@@ -0,0 +1,147 @@
+            android:layout_height="wrap_content"
+            android:clickable="true"
+            android:focusable="true"
+            android:orientation="vertical">
+
+            <org.wikipedia.views.FaceAndColorDetectImageView
+                android:id="@+id/viewArticleImage"
+                android:layout_width="match_parent"
+                android:layout_height="201dp"
+                android:background="?attr/selectableItemBackgroundBorderless" />
+
+            <LinearLayout
+                android:id="@+id/viewArticleContainer"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical"
+                android:paddingStart="16dp"
+                android:paddingEnd="16dp">
+
+                <TextView
+                    android:id="@+id/viewArticleTitle"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="10dp"
+                    android:background="?attr/selectableItemBackgroundBorderless"
+                    android:ellipsize="end"
+                    android:fontFamily="serif"
+                    android:lineSpacingExtra="4sp"
+                    android:maxLines="2"
+                    android:textColor="?attr/primary_text_color"
+                    android:textSize="24sp"
+                    tools:text="Lorem ipsum" />
+
+                <org.wikipedia.views.GoneIfEmptyTextView
+                    android:id="@+id/viewArticleSubtitle"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="24dp"
+                    android:layout_marginEnd="5dp"
+                    android:background="?attr/selectableItemBackgroundBorderless"
+                    android:ellipsize="end"
+                    android:lineSpacingExtra="6sp"
+                    android:maxLines="3"
+                    android:textColor="?attr/secondary_text_color"
+                    android:textSize="14sp"
+                    android:visibility="gone"
+                    tools:text="Lorem ipsum" />
+
+                <View
+                    android:id="@+id/articleDescriptionPlaceHolder1"
+                    android:layout_width="222dp"
+                    android:layout_height="8dp"
+                    android:layout_marginTop="10dp"
+                    android:background="?attr/tab_placeholder_text_color"
+                    android:visibility="gone" />
+
+                <View
+                    android:id="@+id/articleDescriptionPlaceHolder2"
+                    android:layout_width="148dp"
+                    android:layout_height="8dp"
+                    android:layout_marginTop="12dp"
+                    android:background="?attr/tab_placeholder_text_color"
+                    android:visibility="gone" />
```

Comment:
Looks like there has an unnecessary top margin in the "translate" article description.

The title and "source" language description has a large margin between theme.

Reply:
"Looks like there has an unnecessary top margin in the "translate" article description."

Do you mean the space between the call to action button and the extract text? In any case, the problem there is that the extract has html tag at the bottom which is causing those gaps. I was thinking of asking Robin if it is ok, or if we can just use the plain text. A third approach would be to use the plain text and bolden the required portion on our own [least liked by me ]. What do you feel?

Reply:
I saw you adjust the top margin in the description view and that what I commented on.

> the extract has html tag at the bottom which is causing those gaps
Yes, we should use the plain extract and I have confirmed with Robin, now you can notice that the Random article and Featured article are using the plain extract.